index.html

```diff
@@ -4,7 +4,7 @@
     <meta charset="utf-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1">
-    <meta name="description" content="It is an Advisory firm with many decades of experience in the Securities Industry and Investment Consultants in Shares and are dealing in Non Traded / Unlisted / Unquoted Shares">
+    <meta name="description" content="Gandhi Associates is an Advisory firm  in the Securities Industry and Investment Consultants in Shares.">
```

Comment:
Update content

Gandhi Associates is an advisory firm in the securities industry and investment consultants in shares.